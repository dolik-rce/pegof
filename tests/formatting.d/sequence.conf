input formatting.d/sequence.peg
