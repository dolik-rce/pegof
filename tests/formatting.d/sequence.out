X <- A [0-9]* "_" . . .

A <- [A-Z] [a-z]
